Tools/autotest/param_metadata/param_parse.py

```diff
@@ -81,7 +85,7 @@ def error(str_to_print):
     f = open(vehicle.path+'/Parameters.' + extension)
     p_text = f.read()
     f.close()
-
+    current_file = vehicle.path+'/Parameters.' + extension
```

Comment:
Why not set this before opening the file and then use it for open?

Reply:
yep, that is better ! I will change that